xml/admin_ceph_upgrade.xml

```diff
@@ -633,7 +631,7 @@ ID   CLASS  WEIGHT   TYPE NAME       STATUS  REWEIGHT  PRI-AFF
   </procedure>
  </sect1>
  <sect1 xml:id="upgrade-gateways">
-  <title>Upgrade the Gateways</title>
+  <title>Gateway Upgrades</title>
```

Comment:
```suggestion
  <title>Upgrading &ceph; Gateways</title>
```

Reply:
Not a procedure, it's a section header. The below are all gerunds